applications/new_schema.py

```diff
@@ -122,14 +122,36 @@ def mutate_and_get_payload(cls, root, info, **input):
 class Query:
     berth_application = graphene.relay.Node.Field(BerthApplicationNode)
     berth_applications = DjangoFilterConnectionField(
-        BerthApplicationNode, filterset_class=BerthApplicationFilter
+        BerthApplicationNode,
+        filterset_class=BerthApplicationFilter,
+        statuses=graphene.List(graphene.String),
```

Comment:
can it accept graphene's enums?
![image](https://user-images.githubusercontent.com/24206160/75333387-bdbb7200-588e-11ea-8266-f82543d6f7d1.png)

Reply:
would help avoid checking if such an enum exists 😏

Reply:
Just figured out how 🤯 I was trying passing the enum value as string (in quotes) when it's not needed.

Reply:
yeah, according to GQL specs, if it is part of the operation - it should be without quotation marks, but if it is passed in variables, then it can be in quotation marks.